### PR TITLE
Cleaning RB issues

### PR DESCRIPTION
**Pull request by hernanmd**
- Clean #renameClassVariable:to:in:around: which has no users: https://github.com/pharo-project/pharo/issues/11512
- Clean RBReplacePreview without users in Pharo 11: https://github.com/pharo-project/pharo/issues/11515 
- Enabled the copy package command when right-click over a selected package in a Calypso browser. The SycCopyPackageCommand has a commented in the menu item enabler in its #canBeExecutedInContext:, and it was always returning false.

### CODE REVIEW

File: src/Refactoring-Core/RBNamespace.class.st [hunk 1]
```diff
@@ -667,15 +667,6 @@ RBNamespace >> renameClass: aRBClass to: aSymbol around: aBlock [
 	^ change
 ]
 
-{ #category : #'private - changes' }
```
Comment (Ducasse) [inline src/Refactoring-Core/RBNamespace.class.st:670]: Why do you remove it?

**Comment by Ducasse** [comment]
I think that balsa wrote a better preview so we can remove this one indeed.
For the copy this is cool. 
I would like to have the rename with regexp available.

File: src/Refactoring-Core/RBNamespace.class.st (hunk 1 above)
Comment (hernanmd) [reply src/Refactoring-Core/RBNamespace.class.st:670]: Because you wanted to #11512 :)

**Comment by MarcusDenker** [comment]
- [CI] System variable environment on windows does not use adequate encodings #11407

**Comment by hernanmd** [comment]
> I think that balsa wrote a better preview so we can remove this one indeed. For the copy this is cool. I would like to have the rename with regexp available.

I just pushed this one:

**Comment by Ducasse** [comment]
This is nice. Did you see the Driver concept? so that we do not add interaction in the Commander nor the refactoring?

**Comment by Ducasse** [comment]
I could not see the results of the tests :(

**Comment by hernanmd** [comment]
> I could not see the results of the tests :(

In the RBCopyPackageParametrizedTest #tearDown I remove the package created for test.

```smalltalk
tearDown

	(RBRefactoryChangeFactory instance removePackageNamed: 'RefactoringSmallPackageForTest')
		execute.
	super tearDown
```

I'll delete it for the next commit

**Comment by hernanmd** [comment]
> I could not see the results of the tests :(

I didn't see it, some link would be useful.

**Comment by Ducasse** [comment]
I meant in the CI. Apparently it was doing a 404 or something like that.

**Comment by Ducasse** [comment]
It seems to work now. just super slow.

**Comment by Ducasse** [comment]
osx-64 / Tests-osx-64 / testAndMakeSureSuperTearDownIsCalledAsLastMessageInTearDownMethodsOfTestCases – MacOSX64.ReleaseTests.CleanCode.ProperlyImplementedSUnitClassesTest
<1s
unix-64 / Tests-unix-64 / testAndMakeSureSuperTearDownIsCalledAsLastMessageInTearDownMethodsOfTestCases – Unix64.ReleaseTests.CleanCode.ProperlyImplementedSUnitClassesTest
<1s
Existing failures - 8
windows-64 / Tests-windows-64 / testAndMakeSureSuperTearDownIsCalledAsLastMessageInTearDownMethodsOfTestCases – Windows64.ReleaseTests.CleanCode.ProperlyImplementedSUnitClassesTest

File: src/Refactoring2-Transformations-Tests/RBCopyPackageParametrizedTest.class.st [hunk 2]
```diff
@@ -16,59 +16,102 @@ RBCopyPackageParametrizedTest >> constructor [
...
+	model := self defaultNamespace new.
+	self
+		createPackageNamed: 'RefactoringSmallPackageForTest'
+		withClasses: #('RBFTTest1' 'RBFTTest2')
+]
+
+{ #category : #running }
+RBCopyPackageParametrizedTest >> tearDown [
+
+	super tearDown.
```
Comment (Ducasse) [inline src/Refactoring2-Transformations-Tests/RBCopyPackageParametrizedTest.class.st]: super tearDown should be the last expression.

Comment (hernanmd) [reply src/Refactoring2-Transformations-Tests/RBCopyPackageParametrizedTest.class.st]: Ok, fix in https://github.com/pharo-project/pharo/commit/e2dc9b8466c9745af2f71b97a859c170334b27fb

**Pull request merged**